sdk/storage/Azure.Storage.Blobs/src/Generated/BlobAcquireLeaseHeaders.cs

```diff
@@ -20,7 +20,7 @@ public BlobAcquireLeaseHeaders(Response response)
         }
         /// <summary> Returns the date and time the blob was last modified. Any operation that modifies the blob, including an update of the blob&apos;s metadata or properties, changes the last-modified time of the blob. </summary>
         public DateTimeOffset? LastModified => _response.Headers.TryGetValue("Last-Modified", out DateTimeOffset? value) ? value : null;
-        /// <summary> Uniquely identifies a blobs&apos;s lease. </summary>
+        /// <summary> Uniquely identifies a blobs&apos; lease. </summary>
```

Comment:
is this a correct change?

Reply:
@tasherif-msft

Reply:
Yes, this is a correct change, its plural possessiveness so rather than "blobs's lease" it supposed to be "blobs' lease".